setup.py

```diff
@@ -13,9 +14,10 @@
 
 requirements = ['koji', 'toolchest']
 
-setup_requirements = ['pytest-runner', ]
+needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
+setup_requirements = ['pytest-runner'] if needs_pytest else []
```

Comment:
We should just use `setup_requirements = ['pytest-runner']` here. We might be able to move this to the test_requirements, but I seem to remember having issues when I did that.

Reply:
pytest-runner is for setup.py support so you can run python setup.py pytest  ... 
it also enable other options for functionality of pytest.

Reply:
@yazug I agree. I am saying the other logic around it is not needed so lets drop it down to just `setup_requirements = ['pytest-runner']` instead of the `needs_pytest` and the `if/else` statement.